src/sales/shopping-assisted-cart-manage.md

```diff
@@ -63,17 +63,16 @@ _Shopping Cart in in the customer account_
    ![]({% link images/images-ee/customer-account-manage-cart-update-cart.png %}){: .zoom}
    _Cart Updated_
 
-### Add a configurable product
+### Add a product with configuration
+
+There are 4 types of products that need to be configured before adding to the cart: `Bundle Product`, `Configurable Product`, `Grouped Product`.
```

Comment:
We spell out numbers under 10 as a general tech writing style rule and there are only three named here

```suggestion
There are three types of products that need to be configured before adding to the cart: `Bundle Product`, `Configurable Product`, and `Grouped Product`.
```

Reply:
Thanks, done ✅